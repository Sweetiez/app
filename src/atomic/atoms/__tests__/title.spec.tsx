import * as React from 'react';
import renderer from 'react-test-renderer';
import Title from '../title';

describe('Title', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Title title={'test'} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('renders with specified size', () => {
    const tree = renderer.create(<Title title={'test'} size={34} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
