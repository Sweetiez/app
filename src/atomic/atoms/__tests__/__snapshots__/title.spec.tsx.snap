// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correctly 1`] = `
<Text
  size={32}
  style={
    Array [
      Object {
        "color": "black",
        "fontFamily": "Pompiere-Regular",
        "fontSize": 32,
        "textAlign": "center",
      },
    ]
  }
>
  test
</Text>
`;

exports[`Title renders with specified size 1`] = `
<Text
  size={34}
  style={
    Array [
      Object {
        "color": "black",
        "fontFamily": "Pompiere-Regular",
        "fontSize": 34,
        "textAlign": "center",
      },
    ]
  }
>
  test
</Text>
`;
